inductiva/_cli/cmd_resources/list.py

```diff
@@ -8,14 +8,23 @@
 def list_machine_types_available(unused_args):
     """List all available machines types."""
 
-    print("Available machine types\n")
-    print("machine-type: [cores-available]")
-    for (machine_type,
-         cores) in resources.machine_types.AVAILABLE_MACHINES.items():
-        cores_str = ", ".join(str(core) for core in cores)
-        print(f"{machine_type}: [{cores_str}]")
-
-    print("\n E.g. of machine-type: c2-standard-8\n")
+    resources_available = resources.machine_types.get_available()
+
+    # Fetch resources for each provider
+    print()
+    for _, provider_resources in resources_available.items():
```

Comment:
If we don't use the key why not?
```suggestion
    for provider_resources in resources_available.values():
```